backend/agent.py

```
@@ -200,41 +202,44 @@ async def stream_chat(self, user_query: str) -> Generator[Dict[str, str], None,
+                        model_path=self.model_path,
+                        raw=True,
+                        system_instruction=self.system_prompt,
+                        stop=["Observation:", "Question:"],
+                        max_tokens=self.max_tokens,
+                        temperature=0.1,
+                    ),
+                    timeout=self.step_timeout,
                 )
 
                 if response_text.startswith("Error"):
                     raise Exception(response_text)
```

Ruff (0.15.15)

[warning] Abstract `raise` to an inner function

(TRY301)